web/src/shared/components/ourdna/BarChart.tsx

```diff
@@ -0,0 +1,251 @@
+import * as React from 'react'
+import { Card, Container } from 'semantic-ui-react'
+import * as d3 from 'd3'
```

Comment:
Goodness D3 is so clunky 😂 
@milo-hyben, just pulling you here for a consult to see if you have any advice on how to make this a little sleeker

Reply:
Yes, there are few ways to use d3. Most React like is implemented in the StackedAreaByDateChart. Then second close is StackedBarChart.
The rest of charts are more of pure JS than React and we should refactor at some stage.
Basically whenever we need to clear content / reseting SVG in the code, it is a bit of smell not using it React way.
Happy to jump on call and explain in details.